packages/base-types/src/models/version/index.ts

```diff
@@ -57,7 +62,7 @@ export interface Model<_PlatformData extends PlatformData, Command extends BaseC
   domains?: Domain[];
   folders?: Record<string, Folder>;
   variables: Variable[];
-  prototype?: Prototype<Command, Locale>;
+  prototype?: Prototype<Command, Locale, SurveyContextExt>;
```

Comment:
Some boilerplate here to allow us to specify the `SurveyContextExt` in `SurveyContext` from an enclosing `Version.Model` type.